src/main/scala/io/iohk/ethereum/validators/BlockHeaderValidator.scala

```diff
@@ -15,13 +15,19 @@ object BlockHeaderValidatorImpl {
   val GasLimitBoundDivisor: Int = 1024
   val MinGasLimit: BigInt = 5000 //Although the paper states this value is 125000, on the different clients 5000 is used
   val MaxGasLimit = Long.MaxValue // max gasLimit is equal 2^63-1 according to EIP106
+  val MaxPowCaches: Int = 2 // maximum number of epochs for which PoW cache is stored in memory
+
+  case class PowCacheData(cache: Array[Int], dagSize: Long)
```

Comment:
Detail: I'd rather not make it a **case** class since it has a mutable element (default impls of `equals` and `hashCode` are invalid)